assets/sass/components/_badges-tags.scss

```diff
@@ -0,0 +1,12 @@
+a.govuk-link {
+  font-weight: 400 !important;
```

Comment:
I don't think you need the `!important` here - it seems to work without (at least, for me, locally) - would you mind trying without? It's best to try and avoid using `!important` where possible as it can lead to some confusing behaviour when trying to add new styles etc.